calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/calcite/adapter/DocumentDbRules.java

```diff
@@ -1260,28 +1387,51 @@ private static SqlBuilder toSql(SqlBuilder buf, RexLiteral rex) {
      */
     @Getter
     static class Operand {
-        private final String value;
+        private final String aggregationValue;
```

Comment:
Renamed value to aggregationValue and added queryValue. aggregationValue uses aggregation operators while queryValue can only have query operators. Not sure if these are good names 😅